### PR TITLE
change measurements table to measurement_type and add column

### LINKED ISSUE #134: Restructure - measurement type table
Create and link a measurement type table

### LINKED ISSUE #151: Change the name of Measurements table
As discussed in #134, suggest changing the name of table ```measurements``` to ```measurement_type``` to better represent the content of the table.

### PR DESCRIPTION
**Pull request by aaarendt**
Closes #134 
Closes #151

### CODE REVIEW

**Comment by jomey** [comment]
FYI - I was wrong about the 'auto close' of linked issues.
You need to use "Closes #XXX" instead of "Fixes". The latter only references, while the former automatically closes it.

**Review by jomey — APPROVED** [review]

File: snowexsql/tables/measurement_type.py [hunk 1]
```diff
@@ -8,10 +8,12 @@ class MeasurementType(Base):
...
+    __tablename__ = 'measurement_type'
 
-    name = Column(String())
+    name = Column(Text)
     units = Column(Text)
+    derived = Column(Boolean, default=False)
+
 
 
 class HasMeasurement:
```
Comment (jomey) [inline snowexsql/tables/measurement_type.py:19]: For consistency I think we should also rename this class to `HasMeasurementType`

File: snowexsql/tables/measurement_type.py [hunk 2]
```diff
@@ -21,7 +23,7 @@ class HasMeasurement:
 
     @declared_attr
     def measurement_id(cls):
-        return Column(Integer, ForeignKey('public.measurements.id'),
+        return Column(Integer, ForeignKey('public.measurement_type.id'),
                       index=True)
 
     @declared_attr
```
Comment (jomey) [inline snowexsql/tables/measurement_type.py:29]: Also change this definition to 
```python
def measurement_type
```

Comment (aaarendt) [reply snowexsql/tables/measurement_type.py:29]: assuming here you mean
```python
def measurement_type_id
```
for consistency with other foreign key names.

**Pull request merged**